vmdb/spec/controllers/host_controller_spec.rb

```diff
@@ -2,6 +2,20 @@
 
 describe HostController do
   context "#button" do
+    render_views
+
+    it "doesn't break" do
+      set_user_privileges
+      h1 = FactoryGirl.create(:host)
+      h2 = FactoryGirl.create(:host)
+      session[:host_items] = [h1.id, h2.id]
+      session[:settings] = {:views     => {:host => 'grid'},
+                            :display   => {:quad_truncate => 'f'},
+                            :quadicons => {:host => 'foo'}}
```

Comment:
@tenderlove it would go a long way to put a proxy object in front of session, `@edit`, `@sb`, etc. that we can use for recording interactions much in the same way we do with VCR in order to test this code.  We had to do this manually and makes testing UI code very hard :scream: